lib/domain/model/verification/name_with_space_only_validator.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/domain/app_state/validator/verify_name_view_state.dart';
+import 'package:fluffychat/domain/exception/verify_name_exception.dart';
+import 'package:fluffychat/domain/model/verification/new_name_request.dart';
+import 'package:fluffychat/domain/model/verification/validator.dart';
+
+class NameWithSpaceOnlyValidator extends Validator<NewNameRequest> {
+  @override
+  Either<Failure, Success> validate(NewNameRequest value) {
+    if (value.value != null &&
+        value.value!.isNotEmpty &&
```

Comment:
`isNotEmpty` or `isEmpty` ?

Reply:
Could you help me explain for this case?

Reply:
To verify if the given string contains only spaces,  it needs to be not null and not empty (before trim) but after trimming the spaces it needs to be empty